src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,49 @@
+import { FC, memo } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+export interface TodoListProps {
+  todos: Todo[],
+  onDeleteTodo: (todoId: number) => void;
+  tempTodo: Todo | null,
+  onDeletingTodoIds: number[],
+  updateTodo:(
+    todoId: number,
+    fieldsToUpdate: Partial<Pick<Todo, 'title' | 'completed'>>
+  ) => Promise<void>,
```

Comment:
the function `updatingTodo` from `App.tsx` doesn't return Promise, it's just void
```suggestion
  ) => void,
```

Reply:
It's different function